app/controllers/pages/address_settings_controller.rb

```diff
@@ -3,20 +3,20 @@ def new
     uk_address = session.dig(:page, :answer_settings, :input_type, :uk_address)
     international_address = session.dig(:page, :answer_settings, :input_type, :international_address)
     @address_settings_form = Pages::AddressSettingsForm.new(uk_address:, international_address:)
-    @address_settings_path = address_settings_create_path(@form)
-    @back_link_url = type_of_answer_new_path(@form)
-    render address_settings_view
```

Comment:
Think we can get rid of the `address_settings_view` method if we're not using it any more

Reply:
💯 good spot 👏